make/autoconf/jdk-options.m4

```diff
@@ -591,6 +591,22 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_JLINK_OPTIONS],
   AC_SUBST(JLINK_KEEP_PACKAGED_MODULES)
 ])
 
+################################################################################
+#
+# jlink related.
```

Comment:
This part of the comment seems a bit redundant.

Reply:
Updated.